docs-chef-io/content/inspec/resources/aide_conf.md

```diff
@@ -49,13 +49,33 @@ Use the where clause to match a selection_line to one rule or a particular set o
 
 ## Properties
 
-- `conf_path`, `content`, `rules`, `all_have_rule`
+- [`rules`](#rules)
+- [`all_have_rule`](#all_have_rule)
+- [`selection_lines`](#selection_lines)
```

Comment:
I think we can delete all these lists of properties because each property immediately follows the list.

Reply:
Do you mean delete the linking or the whole list? And does that need to be done for all or for this resource only?

Reply:
Yes, the whole list and for all the resources. 
If you look at the build preview, there's a list of the properties immediately followed by the properties themselves. The list is just unnecessary.
https://deploy-preview-5677--chef-inspec.netlify.app/inspec/resources/aide_conf/#properties

Reply:
@lanMadd I did the required changes. Thanks!!